docs/config/events.md

```diff
@@ -49,11 +49,13 @@ Specifically, you need to hook into an event where the service you are running t
 Usage
 -----
 
-It's fairly straightforward to add events to your [Landofile](./lando.md) using the `events` top level config. Note that due to the nature of events eg automating steps that the *user* usually runs all event commands are run as "you" and do not have `sudo` or `root` access.
+It's fairly straightforward to add events to your [Landofile](./lando.md) using the `events` top level config.
 
-### Appserver commands
```

Comment:
Should this entire line be removed?

Reply:
@uberhacker its replaced by `### Default commands`
http://pr-1440-pe426iq-iaimrn624qfk6.us.platform.sh/config/events.html#default-commands